src/main.rs

```diff
@@ -55,15 +57,42 @@ const FIXED_KEY_VARIABLE: &[&str] = &[
     ];
 
 fn main() -> Result<(), io::Error> {
-    let en_path = "../aice-web/langs/en-US.json";
-    let ko_path = "../aice-web/langs/ko-KR.json";
-    let _en_key_list = extract_keys_from_json(en_path)?;
-    let _ko_key_list = extract_keys_from_json(ko_path)?;
-    let web_files = get_files_with_extension("../aice-web/src", "rs")?;
-    let css_files = get_files_with_extension("../aice-web/static", "css")?;
-    let frontary_file_paths = get_files_with_extension("../frontary", "rs")?;
+    //cloning the repo at runtime
+    let repo_manager = match RepoManager::new() {
+        Ok(manager) => manager,
+        Err(e) => {
+            eprintln!("Error creating temp directory: {e}");
+            return Err(e);
+        }
+    };
+
+    let frontary_url = "https://github.com/aicers/frontary.git";
+    let ui_url = "git@github.com:aicers/aice-web.git";
+
+    repo_manager
+        .clone_repo(frontary_url, "frontary")
+        .expect("Failed to clone frontary");
```

Comment:
`expect` should be used in situations where panic will never occur, does this part meet those conditions?
If so, could you please change the message to the reason why panic  never occur.